doc/dev/quick_start.rst

```diff
@@ -40,8 +40,9 @@ PennyLane. However, some of PennyLane's features may not be fully supported yet,
 
 .. warning::
 
-    The supported backend devices are currently ``lightning.qubit``, ``lightning.kokkos``,
-    ``braket.local.qubit``, ``braket.aws.qubit``, and ``oqc.cloud`` but future plans include the addition of more.
+    Not all PennyLane devices currently work with Catalyst. Supported backend devices include
+    ``lightning.qubit``, ``lightning.kokkos``, ``lightning.gpu``, and ``braket.aws.qubit``. For
```

Comment:
lightning.gpu is not supported
```suggestion
    ``lightning.qubit``, ``lightning.kokkos``, and ``braket.aws.qubit``. For
```
Since this list is evolving, it might make sense to remove it from here entirely now that we have the dedicated page, or maybe only list lightning.qubit as an example to point users to our main/default device.

Reply:
Oops 😅

Reply:
I'll keep the list short at qubit, kokkos, and aws, since this covers CPU/GPU/HW which is nice